proto/xudrpc.proto

```diff
@@ -98,6 +105,17 @@ service Xud {
   }
 }
 
+message ChannelBalanceRequest {
+  // The ticker symbol of the currency to query for 
+  string currency = 1 [json_name = "currency"];
+}
+message ChannelBalanceResponse {
+  // Sum of channels balances denominated in satoshis
```

Comment:
What about litoshis? I think you already phrased this like "denominated in satoshis or equivalent" somewhere else.

Reply:
Yeah we could stick to "satoshis or equivalent" throughout. I was also thinking maybe "smallest unit" - but maybe for now "satoshis or equivalent" is good enough and we'll see what it looks like when more chains have lnd support.

Reply:
"smallest unit" could be confusing because of millisatoshis.

Reply:
True, maybe "smallest blockchain unit" or "smallest discrete unit" but that's getting verbose. We'll deal with it when we have support for more than just a couple of currencies on lnd. I'll make it "or equivalent" for now.